documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -163,47 +163,41 @@ Example: [https://balsamiq.cloud/scs7giw/ponj59g/rD5AC](https://balsamiq.cloud/s
 
 ---
 
-## `/applications/[appId]?mode=assignment`
+## `/review/[appId]`
 
-### Application assignment mode
+Overview/Start page for reviews associated with specific Application. User will see different content depending on permissions. For Reviewers who have assigned content to review, they will have the option to "Start" review, which will create a Review in the database and re-direct to the specific review page.
 
-The “home page” of a specific application during assignment of reviewers.
+## `/review/[appId]/[reviewId]`
```

Comment:
This URL is looking odd with 2 number in a sequence... Could we change this to `/applications/[appId]/reviews/[reviewId]` ? I think this would be more related to existing urls ending with `s`. What you guys think? @andreievg @CarlosNZ

Reply:
Yeah, I kind of agree. Happy to change it.

Reply:
Slightly unsure about plural (`\review` vs `\reviews`) but it's a tiny change if we change our mind.

Reply:
I think since we allow more than one review (for different reviewers, but anyway) would be good to use the plural

Reply:
I think both the application and review should be singular, but like the idea

Reply:
Okay, singular across the board (except for `/applications` list) . Done b4886c7